mmv1/third_party/terraform/tests/data_source_google_service_account_jwt_test.go

```diff
@@ -89,6 +99,13 @@ func TestAccDataSourceGoogleServiceAccountJwt(t *testing.T) {
 	serviceAccount := getTestServiceAccountFromEnv(t)
 	targetServiceAccountEmail := BootstrapServiceAccount(t, getTestProjectFromEnv(), serviceAccount)
 
+	staticTime := time.Now()
+
+	// Override the current time with one that will never change, to compare against later.
+	dataSourceGoogleServiceAccountJwtNow = func() time.Time {
+		return staticTime
+	}
+
```

Comment:
I think testing a scenario where the `expires_in` is updated would help me feel like we aren't missing some edge case. Would you mind adding another step in this existing test?